kubernetes/client/ws_client.py

```diff
@@ -46,6 +46,9 @@ def __init__(self, configuration, url, headers):
         if headers and 'authorization' in headers:
             header.append("authorization: %s" % headers['authorization'])
 
+        header.append("Sec-WebSocket-Protocol: %s" %
```

Comment:
I'll add the none check.
Are you ok with keeping the default value in configuration.py or would you rather have it here?